htdocs/core/class/commoninvoice.class.php

```diff
@@ -1999,4 +1999,5 @@ abstract class CommonInvoiceLine extends CommonObjectLine
 	public $fk_user_modif;
 
 	public $fk_accounting_account;
+	public $code_ventilation;
```

Comment:
This seems deprecated. 
It has been replaced with fk_accounting_account.

Reply:
![image](https://github.com/Dolibarr/dolibarr/assets/3624836/ca98e01b-b68c-4818-a54c-028d7dda04c0)